src/_selectors/FullTradeSelectors.js

```diff
@@ -126,17 +126,40 @@ const durationSecHelper = duration => {
     return durationToSecs(d, u);
 };
 
+const shouldBlockExist = (min, max, unit) => {
```

Comment:
Not a good name, no idea what this means ;)
Also we should extract to more general biz logic module, this is not selector specific.